docs/configuration.md

```diff
@@ -0,0 +1,60 @@
+# Configuration
+
+Operator configuration can be modified by editing the SSP Custom Resource (CR).
+
+## SSP Custom Resource (CR)
+
+To activate the operator, you will need to create a Custom Resource (CR).
+An example CR can be found in the file [config/samples/ssp_v1beta2_ssp.yaml](../config/samples/ssp_v1beta2_ssp.yaml).
+
+## Sample Explained
+
+```
+apiVersion: ssp.kubevirt.io/v1beta1
+kind: SSP
+metadata:
+  annotations:
+    ssp.kubevirt.io/vm-console-proxy-enabled: "true" # Used by VM console proxy operand
+    ssp.kubevirt.io/vm-console-proxy-namespace: "kubevirt" # Used by VM console proxy operand
+    kubevirt.io/operator.paused: "false" # Pause operator reconciliation
+  name: ssp-sample
+  namespace: kubevirt
+spec:
+  commonTemplates: # A set of common templates to create KubeVirt Virtual Machines (VMs)
+    namespace: kubevirt
+  templateValidator: # Validating webhook to check templates sent to the cluster
+    replicas: 2
+  featureGates: # New features that are not enabled by default
+    deployTektonTaskResources: true
+  tektonPipelines:
+    namespace: kubevirt
+  tektonTasks:
+    namespace: kubevirt
+```
+
+### Annotations
+
+The annotations are used by VM console proxy operand.
+
+It is required in order to deploy VM console proxy resources, and in which namespace:
+```
+annotations:
+  ssp.kubevirt.io/vm-console-proxy-enabled: "true" # If not set, then by default is false
+  ssp.kubevirt.io/vm-console-proxy-namespace: "kubevirt" # If not set, then default namespace is "kubevirt"
+```
+
+### Feature Gates
+
+The featureGates field is an optional set of optional boolean feature enabler.
+The features in this list are advanced or new features that are not enabled by default.
```

Comment:
missing list?

Reply:
Nope, it means the list in the configuration. I've adjusted it to make it clearer.